src/components/Switch/Switch.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import React, {useState} from 'react';
+import { action } from '@storybook/addon-actions';
+import Switch from './Switch';
+
+export default {
+    title: 'Switch',
+    component: Switch
+}
+
+export const DefaultSwitch = () => (
```

Comment:
The default switch has two identical switches. If their states are different, one is on and the other is off, it is necessary to display two, otherwise, it is enough to display one.

Reply:
Done